app/controllers/administrate/application_controller.rb

```diff
@@ -10,10 +10,13 @@ def index
       resources = apply_collection_includes(resources)
       resources = order.apply(resources)
       resources = resources.page(params[:_page]).per(records_per_page)
-      page = Administrate::Page::Collection.new(dashboard, order: order)
+      page = Administrate::Page::Collection.new(
+        dashboard,
+        resources,
+        order: order,
+      )
```

Comment:
Moving the resources into the `Page::Collection` object is probably a good call, and it bring it in line with the others page objects :+1: